openedx/core/djangoapps/video_config/models.py

```diff
@@ -4,6 +4,7 @@
 from config_models.models import ConfigurationModel
 from django.db.models import BooleanField
 
+from request_cache.middleware import request_cached
```

Comment:
@efischer19 I am also seeing the following on running `paver studio` with this import in place:
```bash
WARNING:xblock.plugin:Unable to load XModuleDescriptor 'video'
Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/xblock/plugin.py", line 146, in load_classes
    yield (class_.name, cls._load_class_entry_point(class_))
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/xblock/plugin.py", line 75, in _load_class_entry_point
    class_ = entry_point.load()
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2296, in load
    return self.resolve()
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2302, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/video_module/__init__.py", line 10, in <module>
    from .video_module import *
  File "/edx/app/edxapp/edx-platform/common/lib/xmodule/xmodule/video_module/video_module.py", line 27, in <module>
    from openedx.core.djangoapps.video_config.models import HLSPlaybackEnabledFlag
  File "/edx/app/edxapp/edx-platform/openedx/core/djangoapps/video_config/models.py", line 8, in <module>
    from request_cache.middleware import request_cached
ImportError: No module named request_cache.middleware
```